code/modules/atmospherics/components/omni_devices/_omni_extras.dm

```diff
@@ -28,6 +23,7 @@
 	var/datum/gas_mixture/air
 	var/list/nodes // lazy list of nodes
 	var/datum/pipe_network/network
+	var/decl/material/gas/filtering //Our filtering gas, if any.
```

Comment:
This is typed as an instance but code seems to use a type, or at least GET_DECL's it.